__tests__/index.test.ts

```diff
@@ -4,8 +4,21 @@ import * as scope from '../src/scope';
 // @ts-ignore
 import { acquireLockSpy, releaseLockSpy } from 'browser-tabs-lock';
 
+jest.mock('browser-tabs-lock');
```

Comment:
not needed - node modules are automatically mocked

Reply:
Not sure where that has come from - I had a conflict here, possibly a dodgy merge.